hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -90,19 +91,46 @@
    * Total number of log records that were compacted by a compaction operation
    */
   @Nullable
-  private Long totalLogRecords;
+  private long totalLogRecords;
 
   /**
-   * Total number of log files that were compacted by a compaction operation
+   * Total number of log files for a file slice with this base fileid
    */
   @Nullable
-  private Long totalLogFiles;
+  private long totalLogFiles;
```

Comment:
should nt these be part of PerfStat too? CommitMetadata should only contain information about the write itself - which this and some of the other metrics are not. Can you move them as well

Reply:
Yeah this is totaLogFiles that were compacted in the write itself, I'll rename this. All others are stats about the write itself and not general stats..